src/store/RelayStore.js

```diff
@@ -171,11 +176,35 @@ var RelayStore = {
     );
   },
 
+  /**
+   * Adds an update to the store and commits it immediately. Returns
+   * The RelayMutationTransaction.
+   */
+  commitUpdate(
+    mutation: RelayMutation,
+    callbacks?: RelayMutationTransactionCommitCallbacks
+  ): RelayMutationTransaction {
+    const transaction = this.applyUpdate(mutation, callbacks);
+    transaction.commit();
+    return transaction;
+  },
+
+  /**
+   * @deprecated
+   *
+   * Method renamed to commitUpdate
+   */
   update(
     mutation: RelayMutation,
     callbacks?: RelayMutationTransactionCommitCallbacks
   ): void {
-    this.applyUpdate(mutation, callbacks).commit();
+    warning(
+      false,
+      'RelayStore.%s is deprecated. Please use RelayStore.%s instead.',
```

Comment:
also, it might me helpful to call out the existing of `applyUpdate`, this is a good opportunity to inform users of the new method. maybe

```
'`Relay.Store.update` is deprecated. Please use `Relay.Store.commitUpdate`' +
' or `Relay.Store.applyUpdate` instead.'
```